docs/data-feeds/index.md

```diff
@@ -34,14 +35,50 @@ See the [Data Feeds Contract Addresses](/docs/reference-contracts/) page for a l
   - [Check the latest answer against reasonable limits](#check-the-latest-answer-against-reasonable-limits)
   - [Check the timestamp of the latest answer](#check-the-timestamp-of-the-latest-answer)
 
-## Retrieve the latest asset prices
+## Types of data feeds
+
+Data feeds provide many different types of data for your applications.
+
+- [Price feeds](#price-feeds) 
+- [Proof of reserve feeds](#proof-of-reserve-feeds)
+- [L2 sequencer uptime feeds](#l2-sequencer-uptime-feeds)
+
+### Price feeds
+
+See the [Data Feeds Contract Addresses](/docs/reference-contracts/) page for a list of networks and proxy addresses.  
 
 Smart contracts often act in real-time on data such as prices of assets. This is especially true in [DeFi](https://defi.chain.link/).
 
 For example, [Synthetix](https://www.synthetix.io/) uses Data Feeds to determine prices on their derivatives platform. Lending and borrowing platforms like [AAVE](https://aave.com/) use Data Feeds to ensure the total value of the collateral.
 
 Data Feeds aggregate many data sources and publish them on-chain using a combination of the [Decentralized Data Model](/docs/architecture-decentralized-model/) and [Off-Chain Reporting](/docs/off-chain-reporting/).
 
+To learn how to use price feeds, see the [Price Feeds](/docs/get-the-latest-price/) documentation.
+
+<div class="remix-callout">
+    <a href="/docs/get-the-latest-price/">Price Feeds</a>
+</div>
+
+### Proof of reserve feeds
```

Comment:
We should use the capitalization "Proof of Reserve" rather than "Proof of reserve" across our docs

Reply:
Fix applied.